src/style/index.css

```diff
@@ -10,7 +10,7 @@ html {
 }
 
 body {
-  font-family: "Open Sans", sans-serif;
+  font-family: 'Inter', 'Open Sans', sans-serif;
```

Comment:
```suggestion
```
This won't be required as we are setting `font-family` to `font-sans` in line 15